R/postprocess.R

```diff
@@ -324,7 +324,7 @@ enw_pp_summary <- function(fit, diff_obs,
 #' posterior <- enw_posterior(fit$fit[[1]], var = "expr_lelatent_int[1,1]")
 #' enw_quantiles_to_long(posterior)
 enw_quantiles_to_long <- function(posterior) {
-  long <- melt(posterior,
+  long <- melt(as.data.table(posterior),
```

Comment:
There's an internal function substitute for `as.data.table` in `R/check.R` - should probably use that here (for other argument checking capabilities on prediction and quantile).

Reply:
I think there's also style guidance for using `package::...` style, so probably should update this here to explicitly use `data.table::melt(...)`? maybe even the `melt.data.table` directly?

Reply:
Yes agree. Look at docs for `coerce_dt` @natemcintosh